e2e_tests/provider_cfg/cryptoauthlib/config_608a.toml

```diff
@@ -0,0 +1,39 @@
+#########################################################################
+# The example config file for atecc608a cryptochip.
+# There must be an I2C bus with a cryptochip soldered, otherwise provider 
+# instantiation fails.
+# Not to be used by github CI.
+#########################################################################
+[core_settings]
+# The CI already timestamps the logs
+log_timestamp = false
+log_error_details = true
+
+# The container runs the Parsec service as root, so make sure we disable root
+# checks.
+allow_root = true
+
+[listener]
+listener_type = "DomainSocket"
+timeout = 200 # in milliseconds
+socket_path = "/tmp/parsec.sock"
+
+[authenticator]
+auth_type = "Direct"
```

Comment:
Are these files supposed to be used for manual testing/playing around with the service locally, or do you intend to present them to admins as a starting place for configuring the service with an ATECC device? If it's the former then you can disregard this comment, but otherwise I'd say `auth_type = "UnixPeerCredentials"` is a better choice.

Reply:
So far, we do not have such far-reaching plans. These files are here to make our life easier when we run tests.